packs/device42/actions/device_name_list.py

```diff
@@ -0,0 +1,66 @@
+from st2actions.runners.pythonrunner import Action
+import requests
+
+
+class DeviceNameList(Action):
+    def run(self, type=None, service_level=None, in_service=None, customer=None, tags=None,
```

Comment:
Most of them are filters and they are optional. Don't know if there is better way to define all of them. I'm thinking about maybe define only mostly used fields (we have this statistic from our application admin interface) and add field like "additional parameters" to send optional and less popular options (something like "env" field). Just not sure if this is good practice.